src/features/NoResultsPage/index.js

```diff
@@ -0,0 +1,16 @@
+import image from "../NoResultsPage/icons/no-results.svg";
+import { Image, ImageContainer, NoResultsText } from "./styled";
+import Container from "../../common/Container/index";
```

Comment:
You can just write `../../common/Container` 🙃

Reply:
🙃